clients/gwtrpc/src/main/java/org/activityinfo/legacy/shared/impl/GetSchemaHandler.java

```diff
@@ -290,6 +291,7 @@ public void onSuccess(SqlTransaction tx, SqlResultSet results) {
                         db.setCountry(countries.get(row.getInt("CountryId")));
                         db.setOwnerName(row.getString("OwnerName"));
                         db.setOwnerEmail(row.getString("OwnerEmail"));
+                        db.setHasPendingTransfer(row.getBoolean("pendingTransfer"));
```

Comment:
Should anyone besides the database owner be able to see the transfer status?

Reply:
Yes let's make this contingent on the requesting user being the owner. It might be useful to return this to the requested owner as well, as we could put in a visual indicator and provide them with the accept/reject links in the ui?